db/migrate/20160414185708_create_container_label_tag_mappings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateContainerLabelTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_label_tag_mappings do |t|
+      t.string :labeled_resource_type
```

Comment:
Actually, in this case, since it related to an actual class, perhaps `labeled_resource_class` might be a better name (but I'm ok with _type too)